level2/level2_Dohyung/src/Application/AddString.java

```diff
@@ -0,0 +1,20 @@
+package Application;
+
+public class AddString {
+    public AddString(String[] splitedParts){
+        addSplitedString(splitedParts);
+    }
+
+
+    public void addSplitedString(String[] splitedParts){
+        int count =0;
+
+        for(String s : splitedParts){
```

Comment:
```suggestion
        for(String part : splitedParts){
```
변수명은 의미있게 짓는 것이 좋습니다.